frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java

```diff
@@ -210,7 +208,7 @@ public ChannelFuture initializeServer(
         if (connector.isSsl()) {
             sslCtx = configManager.getSslContext();
         }
-        b.childHandler(new ServerInitializer(sslCtx, connector.isManagement()));
+        b.childHandler(new ServerInitializer(sslCtx, connector.isManagement(), configManager));
```

Comment:
Why are you passing the config manager around? Wouldn't ConfigManager.getInstance() work?